ftva_lab_data/views_utils.py

```diff
@@ -104,3 +106,23 @@ def get_item_display_dicts(item: SheetImport) -> dict[str, Any]:
         "inventory_info": inventory_info,
         "advanced_info": advanced_info,
     }
+
+
+def get_add_edit_item_fields(form: ItemForm) -> dict[str, list[str]]:
+    """Returns a list of field names to be used in the add/edit item form."""
```

Comment:
This docstring (list of names) doesn't match the return type hint (dict of lists of names).